### DISCUSSION TITLE
Feature Request: Enable Hudi Format

### DISCUSSION THREAD

**Discussion by MacHu-GWU** [duckdb/duckdb#8925] [Ideas]
Hudi, DeltaLake, Iceburg are the three most popular delta lake / upsert framework. Can duckdb support Hudi? Maybe just reading would be good enough.

**Comment by alippai** [comment]
DuckDB usually implements it's own readers/writers. Because of this the new "delta kernel" specification might be more suitable for implementation, at least they have an initiative to standardize the format and API. 

The [delta kernel](https://github.com/delta-io/delta/issues/1783) is still incomplete and it's not stable yet, the only implementation is Java for now. I'm not aware of any native Hudi or Iceberg readers or a stable API defined for them.

Edit: found https://issues.apache.org/jira/browse/HUDI-6242

**Comment by alberttwong** [comment]
Related https://github.com/duckdb/duckdb/discussions/10743
